joomla/joomla-cms
# [5.2] No menu must be selected for menu items batch processing

Pull Request for Issue #43215 #43214 

### Summary of Changes
Batch processing of menu items seems to be possible for all menu items, also if no menu is selected.
This PR removes the condition, that a menu must be selcted. 

### Testing Instructions
The more menus you have and the more options like multilingual, many leveles in the menus, the better.
Mark some menuItems - if you want select different levels, different languages, 
Activate the batch processing in the actions menu.
Change language, access level, 
copy or move the selecten menutems.

Make sure that 
- wrong moves give error messages
- all other moves are performed correctly


### Actual result BEFORE applying this Pull Request
If no menu is selected, a message is displayed
COM_MENUS_SELECT_MENU_FIRST="To use batch processing, please first select a Menu in the manager."


### Expected result AFTER applying this Pull Request
The batch processing is possible, also if no menu is selcted. 


### Link to documentations
Please select:
